osbenchmark/builder/utils/jdk_resolver.py

```diff
@@ -62,7 +62,7 @@ def _get_defined_env_vars(self, host):
     def _resolve_major_from_java_home(self, host, java_home_env_var_name, java_home_env_var_value):
         if java_home_env_var_value:
             major_version = self._major_version(host, java_home_env_var_value)
-            if java_home_env_var_name == "JAVA_HOME" or java_home_env_var_name == "JAVA{}_HOME".format(major_version):
+            if java_home_env_var_name in ("JAVA_HOME", f"JAVA{major_version}_HOME"):
```

Comment:
Nice! Very Pythonic

Reply:
s/o to the linter for forcing this change